python/python_direct/python_translate.cpp

```diff
@@ -1021,6 +1021,33 @@ class PythonTranslator : public OptInConstDispatch {
         {smm_op->out(), out_block_scale_tv, out_gamma_tv});
   }
 
+  // Map SdpaFwdOp to SdpaFwdOpRecord
```

Comment:
What does this mean to map the the `SdpaFwdOp` to the SdpaFwdOpRecord` given that you are not using the `SdpaFwdOpRecord` and replacing its usage?

Reply:
It doesn't mean anything. I'll update the comment or remove it.